controllers/classController.js

```diff
@@ -29,22 +33,32 @@ exports.createClass = async (req, res) => {
     );
 
     if (imageUrl) fs.unlinkSync(image);
+git 
     // Get creator info from DB
     const creatorInfo = await Users.findById({ _id });
+    const reInfo = await resourcePerson.findOne({ userId: _id });
 
     // Create and Save info in DB
     const createClass = await Class.create({
       className,
       description,
       price,
+      duration,
+      curriculum,
+      course: reInfo.course,
```

Comment:
@BigB97 please can you explain what is going here 
why are you saving the curriculum directly in the class collection and still saving it in the curriculum collection..that is more space to be taken

Reply:
This is what I'm trying to do with this curriculum, resource person most input the link of curriculum (since its not save to the DB) when creating the class at the same time save in its own Collection, it will be easy to use like that..tho I'm just seeing there's another way to do it